pilot3.qmd

```diff
@@ -2,10 +2,54 @@
 title: "R Submission Pilot 3"
 ---
 
-## Pilot 3 - Generation of ADaM data sets 
+**Goals**:
 
-**Goal**: Example R based submission contains ADaM data sets generated using R based on the content of the [Pilot 1](pilot1.qmd) submission.
+1.  Deliver a publicly accessible R-based Submission through the eTCD portal
+1.  Expand on the work done in [Submission Pilot 1](pilot1.qmd), by now utilizing R to generate ADaM datasets.
 
-::: {.callout-warning}
-This page is under construction. More information is forthcoming!
-:::
\ No newline at end of file
+**Timeline**: Submitted to FDA on September 11th, 2023
+
+**Data and analysis scope**:
+  
+- 5 TLFs created using R
+- 5 ADaMs created using R
+- SDTM data is taken from CDISC Pilot Study using `xpt` files 
+
+**Links**:
+
+[R Submission Pilot 3 Development Repo](https://github.com/RConsortium/submissions-pilot3-adam)
+
+- Development area for Pilot 3 Team
+- See Closed Issues discussion for how we worked
+- Check out QC Findings
+
+[R Submission Pilot 3 eCTD Submission Package to FDA](https://github.com/RConsortium/submissions-pilot3-adam-to-fda)
+
+- Materials submitted to the FDA via eTCD
+
+[R Submission Pilot 3 Website](https://rconsortium.github.io/submissions-pilot3-adam/main/)
+
+- Breakdown of the structure of the repos and files
+- Renderings of TLFs, datasets, ADRG and Cover Letter
+- Discussions at Conferences and feedback from FDA
+- Links to QC Findings
```

Comment:
Should we refer to this link in the Pilot 3 Website?
https://github.com/RConsortium/submissions-pilot3-adam/wiki 

I do not see this linked in our website currently, which would bring the user to both the `Common Issues` and `QC Findings`.

Reply:
added

Reply:
Thanks, Ben.  After another review, I realized we had already linked to the QC findings under the `R Submission Pilot 3 Development Repo` section.  Removed to avoid duplication.